v2/getting-started.md

```diff
@@ -2,8 +2,42 @@
 
 _Note: These are the steps immediately following the new [installation of the Commerce 2](install.md) module._
 
+Congratulations! You have downloaded Commerce and are now ready to start creating and selling products. YAY! There are just
+a few steps you need to follow before you can create products. Below we detail them out.
```

Comment:
Add a `tl;dr` at bottom and link to the anchor about `drupal commerce:create:store` since it skips all of these steps, maybe?